src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -138,6 +138,13 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// </summary>
         public bool StartFromBeginning { get; set; } = false;
 
+        /// <summary>
+        /// Optional.
+        /// Gets or sets the a date and time to initialize the change feed read operation from.
+        /// <remark>It is best to use UTC date / time format to avoid any discrepencies.</remark>
+        /// </summary>
+        public string StartFromTime { get; set; }
```

Comment:
Are we able to use `DateTime` (or nullable, probably) as the type here? Will that be correctly deserialized?

Reply:
How does that get exposed to other languages like JS or Python if we do it as DateTime or DateTimeOffset?

Reply:
Nevermind... that won't work for an attribute anyway... so it'll have to be a string.